ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.view.js

```diff
@@ -36,7 +36,7 @@ module.exports = Marionette.LayoutView.extend({
         this.listenTo(this.model, 'change:type', this.changeView);
     },
     onRender: function() {
-        if (this.model.get('type') === 'basic' || this.model.get('type') === 'text' || this.model.get('type') === 'new-form') {
+        if (this.model.get('type') === 'basic' || this.model.get('type') === 'text' || this.model.get('type') === 'new-form' || this.model.get('type') === 'newResult') {
```

Comment:
✏️ Small nite, but I see `new-form` and I see `newResult`. Should we remain consistent with the casing?